### PR TITLE
[13.0][MIG] purchase_deposit

### PR DESCRIPTION
**Pull request by kongrattapong**
Standard Migration

cc. @kittiu @ps-tubtim  @dkrimmer84

### CODE REVIEW

**Comment by kongrattapong** [comment]
cc. @kittiu

File: purchase_deposit/wizard/purchase_make_invoice_advance.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
+# Copyright 2019 Elico Corp, Dominique K. <dominique.k@elico-corp.com.sg>
+# Copyright 2019 Ecosoft Co., Ltd., Kitti U. <kittiu@ecosoft.co.th>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).\
```
Comment (ps-tubtim) [inline purchase_deposit/wizard/purchase_make_invoice_advance.py:3]:
```suggestion
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

**Review by ps-tubtim — APPROVED** [review]
Code LGTM. Functional test OK.

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
@kongrattapong can you rebase and run pre-commit again? to fix build error.
Thanks.

**Comment by kongrattapong** [comment]
sorry for late reply, rebase already and Thank you all for review,

**Pull request merged**